install.yml

```diff
@@ -1,13 +1,10 @@
-$schema: http://json-schema.org/draft-07/schema
-$id: https://adobe.io/schemas/app-builder-templates/1
```

Comment:
$schema, $id should only be used in schema declarations

Reply:
No, these are identifiers that show what `install.yml` conforms to. For example, I can have `$id` that is a different version, that should be processed with a different schema version.

Reply:
@shazron ,
It looks like I misunderstood the purpose of `$schema`, `$id` described on https://json-schema.org/learn/getting-started-step-by-step.html#starting

> The [$schema](https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.8.1.1) keyword states that this schema is written according to a specific draft of the standard and used for a variety of reasons, primarily version control.
> The [$id](https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.8.2.1) keyword defines a URI for the schema, and the base URI that other URI references within the schema are resolved against.

It mentions a schema declaration only.
I'll update [the current schema declaration](https://github.com/adobe/aio-lib-console-project-installation/blob/553e01b380bbdd47eb7c4a399e66eaf40185eb7e/schema/template.schema.json#L5-L27) to require `$schema` and `$id`.

I returned back `$schema` and `$id`.